src/Model/Resolver/CategoryTree.php

```diff
@@ -62,35 +64,50 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
             return $value[$field->getName()];
         }
 
-        $rootCategoryId = $this->getCategoryId($args);
+        $rootCategoryParameters = $this->getCategoryParameters($args);
+        $rootCategoryId = $rootCategoryParameters['id'];
         $categoriesTree = $this->categoryTree->getTree($info, $rootCategoryId);
         if (!empty($categoriesTree)) {
             $result = $this->extractDataFromCategoryTree->execute($categoriesTree);
-            return current($result);
+            $category = current($result);
+
+            $active = $rootCategoryParameters['is_active'];
+            return array_merge($category, ['is_active' => $active]);
         }
 
         return null;
     }
 
     /**
      * @param array $args
-     * @return int
+     * @return array
      * @throws GraphQlInputException
      */
-    private function getCategoryId(array $args): int
+    private function getCategoryParameters(array $args): array
     {
         if (isset($args['id'])) {
-            return (int)$args['id'];
+            $categoryFactory = $this->categoryFactory->create();
+            $category = $categoryFactory->load((int)$args['id']);
```

Comment:
Why to load a category one more time? Can this be done when loading a collection? I think this way is not good.